### PR TITLE
Readds mild painkillers, makes them the 'default' painkiller.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
## Description of changes
- Most default painkillers will now be 'mild painkillers', which are lower power, but not narcotic, with a higher overdose threshold.
- Strong painkillers have been added, replacing mild painkillers in some contexts.

## Why and what will this PR improve
The default painkiller being morphine-grade has been pretty lethal in the past.

## Authorship
Done with reference to Baystation painkiller values.

## Changelog
:cl:
tweak: Strong (narcotic) painkillers and mild (non-narcotic) painkillers are now distinct chemicals.
/:cl:

### CODE REVIEW

File: code/modules/hydroponics/seed_datums.dm [hunk 1]
```diff
@@ -1312,19 +1315,6 @@
 	set_trait(TRAIT_WATER_CONSUMPTION, 6)
 	set_trait(TRAIT_NUTRIENT_CONSUMPTION, 0.15)
 
-/datum/seed/algae/bruisegrass
```
Comment (MistakeNot4892) [inline code/modules/hydroponics/seed_datums.dm:1315]: Forgot I snipped this. It's a ValSal thing from ScavStation days.

File: code/modules/reagents/reagent_containers/borghydro.dm [hunk 2]
```diff
@@ -19,10 +19,18 @@
 	var/list/reagent_names = list()
 
 /obj/item/chems/borghypo/surgeon
-	reagent_ids = list(/decl/material/liquid/brute_meds, /decl/material/liquid/oxy_meds, /decl/material/liquid/painkillers)
+	reagent_ids = list(
+		/decl/material/liquid/brute_meds,
+		/decl/material/liquid/oxy_meds,
+		/decl/material/liquid/painkillers
```
Comment (out-of-phaze) [inline code/modules/reagents/reagent_containers/borghydro.dm]:
```suggestion
		/decl/material/liquid/painkillers/strong
```
Please don't let surgeon borgs do surgery on people with just Tylenol.

File: code/modules/reagents/chems/chems_painkillers.dm [hunk 3]
```diff
@@ -95,14 +112,14 @@
 
 /decl/material/liquid/painkillers/affect_overdose(var/mob/living/M)
 	..()
-	if(!narcotic)
-		return
-	M.set_hallucination(120, 30)
 	SET_STATUS_MAX(M, STAT_DRUGGY, 10)
```
Comment (out-of-phaze) [inline code/modules/reagents/chems/chems_painkillers.dm]: Why would overdosing on non-opiate painkillers get you high? Maybe move the druggy effect under `if(narcotic)` below.

File: code/modules/reagents/chems/chems_painkillers.dm [hunk 4]
```diff
@@ -95,14 +112,14 @@
...
 	M.add_chemical_effect(CE_PAINKILLER, pain_power*0.5) //extra painkilling for extra trouble
-	M.add_chemical_effect(CE_BREATHLOSS, breathloss_severity*2) //ODing on opiates can be deadly.
-	if(isboozed(M))
-		M.add_chemical_effect(CE_BREATHLOSS, breathloss_severity*4) //Don't drink and OD on opiates folks
+	M.add_chemical_effect(CE_TOXIN, 1)
```
Comment (out-of-phaze) [inline code/modules/reagents/chems/chems_painkillers.dm]: I don't *think* ODing on opiates is toxic? The breathloss is pretty punishing already, isn't it? I'd either put this behind an `else` after `if(narcotic)` or give it its own separate overdose toxicity var.

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**